features/activity/shutdown/README.md

```diff
@@ -10,12 +10,15 @@ complete with failure, and ignore the cancel. The one which ignores the cancel s
 encounter a hard timeout.
 
 # Detailed spec
-* If a worker is told to begin shutdown, activities are immediately notified via
-  a cancel. It must be possible for the user to determine if this cancel was issued
-  by server, or is the result of worker shutdown
+* When a worker shutdown is initiated, the activity context isn't canceled until shutdown
+  has succeeded or the shutdown timeout has elapsed
```

Comment:
I don't think "until shutdown succeeded" is accurate here, we cancel _before_ shutdown succeeds (in fact, shutdown can't succeed until all activities complete in Core-based SDKs and maybe others). May just be worth removing:

```suggestion
* When a worker shutdown is initiated, the activity context isn't canceled until
  the shutdown timeout has elapsed
```